source/conf.py

```diff
@@ -0,0 +1,34 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "csvcubed"
+copyright = "2023, ONS"
+author = "ONS"
+release = "0.3.5"
```

Comment:
This needs to be automatically updated when we do a release (via one of the GitHub workflows)

Reply:
yeah okay will sort it out